src/meta.ts

```diff
@@ -130,7 +130,7 @@ export class Meta {
     const vraw = this.setValue(
       handlebars.compile(tag.attrs['pattern'])({
         date: function (format) {
-          return moment(currentDate).utc().format(format);
+          return moment(currentDate).tz(tag.attrs['timezone']).format(format);
```

Comment:
Instead of an extra attribute it would be better to add an hash arg to the handlebar func like: https://handlebarsjs.com/guide/expressions.html#helpers-with-hash-arguments

```ts
date: function (format, options) {
  const m = moment(currentDate);
  Object.keys(options.hash).forEach(key => {
    switch (key) {
      case "tz":
        m.tz(options.hash[key]);
        break;
      default:
        throw new Error(`Unknown ${key} attribute`);
    }
  })
  return m.format(format);
}
```

Reply:
@crazy-max Thanks for your review. I have updated.